apps/framework-docs-v2/src/components/mdx/cta-card.tsx

```
@@ -44,104 +45,169 @@ export function CTACard({
     2: "grid-cols-1 md:grid-cols-2",
     3: "grid-cols-1 md:grid-cols-2 lg:grid-cols-3",
     4: "grid-cols-1 md:grid-cols-2 lg:grid-cols-4",
   };
 
+  // Grouped mode - render cards in a single card container with dividers
+  if (grouped) {
+    const childArray = React.Children.toArray(children);
+    return (
+      <div className="not-prose rounded-xl border bg-card text-card-foreground shadow overflow-hidden mt-5 w-full">
+        {childArray.map((child, index) => (
+          <React.Fragment key={index}>
```

Biome (2.3.13)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)